index.html

```diff
@@ -194,6 +194,39 @@ <h6 class="skill skills">Skills</h6>
     </section>
   </main>
   <footer>
+    <section class="contact-page">
+      <div>
+        <h2 class="contact-page-head">I'm always interested in hearing about new projects, so if you'd like to chat
+          please get in touch.</h2>
+      </div>
+      <form class="inputs" action="https://formspree.io/f/xayvdjzb" method="post">
+        <ul class="input-list">
+          <li>
+            <input class="input" type="text" maxlength="30" name="full-name" placeholder="Full name" required>
+          </li>
+          <li>
+            <input class="input" type="email" name="email-address" placeholder="Email address" required>
+          </li>
+          <li>
+            <textarea class="input textarea" placeholder="Enter text here" maxlength="500" required></textarea>
+          </li>
+          <li class="submit-button">
+            <button class="button" type="submit" name="submit" value="send">Get in touch</button>
+          </li>
+        </ul>
+      </form>
+      <ul class="social-contact-page">
+        <li><a href="https://github.com/ahzamir" target="blank"><img class="icon" alt="github"
+              src="images/github2.svg"></a>
+        </li>
+        <li><a href="https://www.linkedin.com/in/ahmad-zamir-yousufi-70603317b/" target="blank"><img class="icon"
```

Comment:
Please ensure that you stick to the design as much as possible (e.g., font, colors, images, tests, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

Your Implementation | Figma Design
:---:|:---:
![image](https://user-images.githubusercontent.com/10332499/160699668-3de3f4e4-2d87-4184-afc3-34b5d8e931c4.png) | ![image](https://user-images.githubusercontent.com/10332499/160699554-8c2977d8-dcc7-48e5-8d37-97e47c5a045d.png)